vue/components/ui/molecules/calendar/calendar.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+    <div class="calendar">
+        <div class="calendar-header">
+            <button-icon
+                icon="chevron-left"
+                icon-color="#1d2631"
+                v-bind:icon-opacity="1"
+                v-bind:icon-stroke-width="3"
+                v-on:click.stop="onLeftClick"
+            />
+            <div class="header-center">
+                {{ headerText }}
+            </div>
+            <button-icon
+                icon="chevron-right"
+                icon-color="#1d2631"
+                v-bind:icon-opacity="1"
+                v-bind:icon-stroke-width="3"
+                v-on:click.stop="onRightClick"
+            />
+        </div>
+        <div class="calendar-content">
+            <table class="calendar-table">
+                <thead class="table-header">
+                    <tr class="row">
+                        <th
+                            class="heading"
+                            v-for="weekLabel in weekLabelsComputed"
+                            v-bind:key="weekLabel"
+                        >
+                            {{ weekLabel }}
+                        </th>
+                    </tr>
+                </thead>
+                <tbody class="table-body">
+                    <tr class="row" v-for="(week, index) in weeks" v-bind:key="index">
+                        <td
+                            class="cell"
+                            v-bind:class="circleClass(week[d - 1].day, week[d - 1].month)"
+                            v-for="d in 7"
+                            v-bind:key="d - 1"
+                            v-on:click="onCellClick(week[d - 1])"
+                        >
+                            <div class="circle">
+                                {{ week[d - 1].day }}
+                            </div>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.calendar {
+    border: 1px solid #e4e8f0;
+    border-radius: 6px 6px 6px 6px;
+    box-shadow: 0px 0px 30px rgba(21, 21, 21, 0.1);
+    display: flex;
+    flex-direction: column;
+    position: absolute;

```

Comment:
Why is this needed?

Reply:
this is better suited in input-date, moving...